### ISSUE TITLE
Moonglade exploration bug

### ISSUE THREAD

**Issue by 186tsmmm** [chromiecraft/chromiecraft#3990]
### What client do you play on?

enUS

### Faction

Horde

### Content Phase:

60

### Current Behaviour

Walking in and around Lake Elune'ara doesn't trigger the exploration credit and achievement
![Screenshot_5](https://user-images.githubusercontent.com/17478149/186488252-a8bb37c5-5626-48d7-ae89-b900717230e7.png)
.

### Expected Blizzlike Behaviour

Walking in and around Lake Elune'ara triggers the exploration credit and achievement.

### Source

This is a pretty straightforward and minor bug, I really don't think this is necessary. 

### Steps to reproduce the problem

1. Go to Moonglade.
2. Nothing happens.
3. Repeat.

### AC rev. hash/commit

https://github.com/chromiecraft/azerothcore-wotlk/commit/e0b7c7d0d3b1ea65860439add2c676b3e18f8f8b

### Operating system

win10

### Modules

- [mod-ah-bot](https://github.com/azerothcore/mod-ah-bot)
- [mod-bg-item-reward](https://github.com/azerothcore/mod-bg-item-reward)
- [mod-cfbg](https://github.com/azerothcore/mod-cfbg)
- [mod-chat-transmitter](https://github.com/azerothcore/mod-chat-transmitter)
- [mod-chromie-xp](https://github.com/azerothcore/mod-chromie-xp)
- [mod-cta-switch](https://github.com/azerothcore/mod-cta-switch)
- [mod-desertion-warnings](https://github.com/azerothcore/mod-desertion-warnings)
- [mod-dmf-switch](https://github.com/azerothcore/mod-dmf-switch)
- [mod-duel-reset](https://github.com/azerothcore/mod-duel-reset)
- [mod-eluna](https://github.com/azerothcore/mod-eluna)
- [mod-ip-tracker](https://github.com/azerothcore/mod-ip-tracker)
- [mod-low-level-arena](https://github.com/azerothcore/mod-low-level-arena)
- [mod-low-level-rbg](https://github.com/azerothcore/mod-low-level-rbg)
- [mod-multi-client-check](https://github.com/azerothcore/mod-multi-client-check)
- [mod-progression-system](https://github.com/azerothcore/mod-progression-system)
- [mod-pvp-titles](https://github.com/azerothcore/mod-pvp-titles)
- [mod-pvpstats-announcer](https://github.com/azerothcore/mod-pvpstats-announcer)
- [mod-queue-list-cache](https://github.com/azerothcore/mod-queue-list-cache)
- [mod-rdf-expansion](https://github.com/azerothcore/mod-rdf-expansion)
- [mod-transmog](https://github.com/azerothcore/mod-transmog)
- [mod-weekend-xp](https://github.com/azerothcore/mod-weekend-xp)
- [mod-instanced-worldbosses](https://github.com/nyeriah/mod-instanced-worldbosses)
- [lua-carbon-copy](https://github.com/55Honey/Acore_CarbonCopy)
- [lua-exchange-npc](https://github.com/55Honey/Acore_ExchangeNpc)
- [lua-custom-worldboss](https://github.com/55Honey/Acore_CustomWorldboss)
- [lua-level-up-reward](https://github.com/55Honey/Acore_LevelUpReward)
- [lua-recruit-a-friend](https://github.com/55Honey/Acore_RecruitAFriend)
- [lua-send-and-bind](https://github.com/55Honey/Acore_SendAndBind)
- [lua-temp-announcements](https://github.com/55Honey/Acore_TempAnnouncements)
- [lua-zonecheck](https://github.com/55Honey/acore_Zonecheck)
- [lua-zone-debuff](https://github.com/55Honey/Acore_ZoneDebuff)

### Customizations

None

### Server

ChromieCraft

**Comment by Sneften** [comment]
Hey! I am unable to recreate the issue.
Can you give me some additional information? From where did you enter moonglade?

I got the achivment entering Moonglade both from the cave leading to Felwood and Nighaven camp.



https://user-images.githubusercontent.com/103856689/186622945-0aefcecb-6938-4e94-bb0c-9c9edef463c2.mp4

**Comment by Rubadubscrub** [comment]
It was bugged a while back. i had the same issue in blasted lands.

submit a support ticket on the Chromie Discord so they can run an admin command to recheck

**Comment by 186tsmmm** [comment]
Will do, thanks!

**Comment by Gultask** [comment]
Indeed. This issue is fixed by the command `.achievement checkall` available to Game Masters! This issue should have been fixed by now after resolving issue with sent packets.

**Comment by 186tsmmm** [comment]
Hi, thanks for getting back to me.
I entered from the cave leading to Felwood. If I remember correctly, (and
this was a bit ago) when I entered the zone, I did get the "Lake Elune'ara"
prompt on the screen and experience for exploring it. It is also explored
on the map. It just never triggered the achievement. I didn't even notice
it right away. I tried going back and running around, in and out, several
times after that, and it never worked. I realize this is wanting for more
detail, I'm sorry I can't be more helpful.
